.results.json

```diff
@@ -0,0 +1 @@
curred while loading ./spec/controllers/songs_controller_spec.rb.\n\u001b[31mFailure/Error:\u001b[0m\n\u001b[31m  \u001b[0m\u001b[1;34;4mRSpec\u001b[0m.describe \u001b[1;34;4mSongsController\u001b[0m, \u001b[35mtype\u001b[0m: \u001b[33m:controller\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[32mlet\u001b[0m(\u001b[33m:valid_attributes\u001b[0m) \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m      \u001b[0m{\u001b[0m\n\u001b[31m        \u001b[0m\u001b[35mtitle\u001b[0m: \u001b[31m\u001b[1;31m\"\u001b[0m\u001b[31mTalisman\u001b[1;31m\"\u001b[0m\u001b[31m\u001b[0m,\u001b[0m\n\u001b[31m        \u001b[0m\u001b[35martist_name\u001b[0m: \u001b[31m\u001b[1;31m\"\u001b[0m\u001b[31mAir\u001b[1;31m\"\u001b[0m\u001b[31m\u001b[0m,\u001b[0m\n\u001b[31m        \u001b[0m\u001b[35mrelease_year\u001b[0m: \u001b[1;34m2007\u001b[0m,\u001b[0m\n\u001b[31m        \u001b[0m\u001b[35mreleased\u001b[0m: \u001b[1;36mtrue\u001b[0m,\u001b[0m\n\u001b[31m        \u001b[0m\u001b[35mgenre\u001b[0m: \u001b[31m\u001b[1;31m\"\u001b[0m\u001b[31mPost-Rock\u001b[1;31m\"\u001b[0m\u001b[31m\u001b[0m\u001b[0m\n\u001b[31m      \u001b[0m}\u001b[0m\n\u001b[31m    \u001b[0m\u001b[32mend\u001b[0m\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mNameError:\u001b[0m\n\u001b[31m  uninitialized constant SongsController\u001b[0m\n\u001b[36m# ./spec/controllers/songs_controller_spec.rb:3:in `\u003ctop (required)\u003e'\u001b[0m\n","\nAn error occurred while loading ./spec/models/song_spec.rb.\n\u001b[31mFailure/Error:\u001b[0m\n\u001b[31m  \u001b[0m\u001b[1;34;4mRSpec\u001b[0m.describe \u001b[1;34;4mSong\u001b[0m, \u001b[35mtype\u001b[0m: \u001b[33m:model\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[32mlet\u001b[0m(\u001b[33m:valid_attributes\u001b[0m) \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m      \u001b[0m{\u001b[0m\n\u001b[31m        \u001b[0m\u001b[35mtitle\u001b[0m: \u001b[31m\u001b[1;31m\"\u001b[0m\u001b[31mTalisman\u001b[1;31m\"\u001b[0m\u001b[31m\u001b[0m,\u001b[0m\n\u001b[31m        \u001b[0m\u001b[35martist_name\u001b[0m: \u001b[31m\u001b[1;31m\"\u001b[0m\u001b[31mAir\u001b[1;31m\"\u001b[0m\u001b[31m\u001b[0m,\u001b[0m\n\u001b[31m        \u001b[0m\u001b[35mrelease_year\u001b[0m: \u001b[1;34m2007\u001b[0m,\u001b[0m\n\u001b[31m        \u001b[0m\u001b[35mreleased\u001b[0m: \u001b[1;36mtrue\u001b[0m,\u001b[0m\n\u001b[31m        \u001b[0m\u001b[35mgenre\u001b[0m: \u001b[31m\u001b[1;31m\"\u001b[0m\u001b[31mPost-Rock\u001b[1;31m\"\u001b[0m\u001b[31m\u001b[0m\u001b[0m\n\u001b[31m      \u001b[0m}\u001b[0m\n\u001b[31m    \u001b[0m\u001b[32mend\u001b[0m\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mNameError:\u001b[0m\n\u001b[31m  uninitialized constant Song\u001b[0m\n\u001b[36m# ./spec/models/song_spec.rb:3:in `\u003ctop (required)\u003e'\u001b[0m\n"],"examples":[],"summary":{"duration":0.000473,"example_count":0,"failure_count":0,"pending_count":0,"errors_outside_of_examples_count":2},"summary_line":"0 examples, 0 failures, 2 errors occurred outside of examples"}
```

Comment:
Whats the story with this file?

Reply:
Ahh ill remove it. This is the file that generates when you run `learn`.

Reply:
Oh! Ok. Thanks!